schemas/model_link.js

```diff
@@ -0,0 +1,22 @@
+const { Decimal128 } = require("bson");
+const { link } = require("fs");
+const mongoose = require("mongoose");
+
+let Link = new Schema(
+    {
+      ZoomLink: {
+        type: String
+      },
+      ClassPage: {
+        type: String
+      },
+      PiazzaLink: {
+        type: String
+      },
+      OtherLink: {
```

Comment:
Is it possible to create an array of "other links"?

Reply:
Yes, it is possible. Please take a look at the change I made.